modules/perform.cpp

```diff
@@ -90,13 +90,15 @@ class CPerform : public CModule {
 	MODCONSTRUCTOR(CPerform) {
 		AddHelpCommand();
 		AddCommand("Add",     static_cast<CModCommand::ModCmdFunc>(&CPerform::Add),
-			"<command>");
+			"<command>", "Add a command to be sent to the server on connect.");
 		AddCommand("Del",     static_cast<CModCommand::ModCmdFunc>(&CPerform::Del),
-			"<nr>");
-		AddCommand("List",    static_cast<CModCommand::ModCmdFunc>(&CPerform::List));
-		AddCommand("Execute", static_cast<CModCommand::ModCmdFunc>(&CPerform::Execute));
+			"<number>", "Delete command <number> in the output of 'List'.");
```

Comment:
Would it be better as "Delete command <number> from the list."? I'm open to suggestions, but only saying why it looks weird isn't particularly helpful. It leads to spending hours going back and forth over what I think you meant.